dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/common/internal/resources/OidcCommonMessages.nlsprops

```diff
@@ -128,8 +128,8 @@ JWK_ENDPOINT_MISSING_ERR.useraction=Set the jwkEndpointUrl attribute to the URL
 #OIDC_FAILED_RUN_AS_SUBJCET.explanation=An exception occurred while getting the RunAsSubject.
 #OIDC_FAILED_RUN_AS_SUBJCET.useraction=Make sure the server is configured and started properly.
 
-#0=OIDC client ID, 1=Current clock time, 2=Token expiration time, 3=Token issued at time
-OIDC_JWT_VERIFY_STATE_ERR=CWWKS1773E: Validation failed for the token requested by [{0}] because the token is outside of its valid range. This might have been caused by either the current time [{1}] being after the token expiration time [{2}] or the issue time [{3}] being too far away from the current time [{1}].
+#0=OIDC client ID, 1=sub claim of the token, 2=Current clock time, 3=Token expiration time, 4=Token issued at time
+OIDC_JWT_VERIFY_STATE_ERR=CWWKS1773E: Validation failed for the token requested by [{0}] for the user [{1}] because the token is outside of its valid range. This might have been caused by either the current time [{2}] being after the token expiration time [{3}] or the issue time [{4}] being too far away from the current time [{2}].
```

Comment:
suggested edit

 Validation failed for the token requested by {0} for the {1} user because the token is outside of its valid range. This error occurs either because the {2} current time is after the {3} token expiration time or because the {4} issue time is too far away from the {2} current time.